qiskit/backends/ibmq/ibmqprovider.py

```diff
@@ -67,9 +73,7 @@ def aliased_backend_names(self):
             }
 
     @classmethod
-    def _authenticate(cls, token, url,
-                      hub=None, group=None, project=None, proxies=None,
-                      verify=True):
+    def _authenticate(cls, token, url, proxies=None, verify=True):
```

Comment:
Do you think it would be a good idea doing a similar change to `IBMQProvider.__init__`, removing the arguments? Even if the class is technically public, if the converting from the old format to the new one is handled in the upper layers, it might make sense to do so and specially might be the best time to do it (adding a note in the docstring).

Reply:
There are places, for example in the tests, were the Provider is called directly.  That is why I left it there, but with the warning, so I do not break something unintentionally.